### PR TITLE
Fix: Downgrade react-icheck to v0.3.4

### PR DESCRIPTION
**Pull request by OndrejCholeva**
React-icheck rewrote thier release history, so v0.3.3 is no more available, so latest version was installed instead. That caused E2E to break as well as issue from PR #361 to reappear.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,7 +85,7 @@
     "react-bootstrap": "^0.29.5",
     "react-datepicker": "^0.29.0",
     "react-hot-loader": "^1.3.0",
-    "react-icheck": "=0.3.3",
+    "react-icheck": "=0.3.4",
```
Comment (bchew) [inline package.json]: `=` is actually not needed https://docs.npmjs.com/misc/semver

Comment (OndrejCholeva) [inline package.json]: I've read that, but decided to go with minimum change. I am fine to remove it if we fell it is easier to read.

**Comment by omgaz** [comment]
LGTM

Make sure you've:
1. rebased with master
2. ran `npm i`
3. ran `npm run dist`

**Comment by OndrejCholeva** [comment]
@omgaz I've spent last 30 minutes doing that (+tests), but still only 95% confident about it 🙈

File: package.json (hunk 1 above)
Comment (bchew) [inline package.json]: Considering there are other places in the file where this is done, perhaps leave it as is. Could probably be done as a separate PR if warranted..

**Comment by lteacher** [comment]
LGTM!

**Comment by OndrejCholeva** [comment]
@omgaz is there anything more I should do?
```
(running of freshmost master)
n 4.6.0
rm -f node_modules
npm i
npm run dist
npm run test
```

If not, please merge.

**Comment by omgaz** [comment]
merging 🚀

**Pull request merged**